http-client/src/main/java/com/facebook/airlift/http/client/thrift/ThriftProtocol.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.airlift.http.client.thrift;
+
+public enum ThriftProtocol
+{
+    BINARY, COMPACT, FB_COMPACT
```

Comment:
There's `com.facebook.drift.transport.netty.codec.Protocol` enum that has the `createProtocol` factory method. You can reuse it.